arm-timeseriesinsights/2017-02-28-preview/Swagger/timeseriesinsights.json

```diff
@@ -0,0 +1,1780 @@
+            "schema": {
+              "$ref": "#/definitions/ReferenceDataSetResource"
+            }
+          },
+          "404": {
+            "description": "The subscription, resource group, environment, or reference data set could not be found."
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "ReferenceDataSets"
+        ],
+        "operationId": "ReferenceDataSets_Update",
+        "x-ms-examples": {
+          "ReferenceDataSetsUpdate": { "$ref": "../examples/ReferenceDataSetsPatchTags.json" }
+        },
+        "description": "Updates the reference data set with the specified name in the specified subscription, resource group, and environment.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ReferenceDataSetNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "referenceDataSetUpdateParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReferenceDataSetUpdateParameters"
+            },
+            "description": "Request object that contains the updated information for the reference data set."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The reference data set definition was successfully updated and is in the response.",
+            "schema": {
+              "$ref": "#/definitions/ReferenceDataSetResource"
+            }
+          },
+          "404": {
+            "description": "The subscription, resource group, environment, or reference data set could not be found."
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ReferenceDataSets"
+        ],
+        "operationId": "ReferenceDataSets_Delete",
+        "x-ms-examples": {
+          "ReferenceDataSetsDelete": { "$ref": "../examples/ReferenceDataSetsDelete.json" }
+        },
+        "description": "Deletes the reference data set with the specified name in the specified subscription, resource group, and environment",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ReferenceDataSetNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
204 means attempt to delete a reference data set that does not exist. 200 means the reference data set was successfully deleted.